examples/pages/google/search.js

```diff
@@ -9,28 +9,15 @@ const searchCommands = {
   }
 };
 
-const consentModal = '[aria-modal="true"]';
-
 module.exports = {
   url: 'https://google.no',
   commands: [
     searchCommands
   ],
 
-  sections: {
-    consentModal: {
-      selector: consentModal,
-      elements: {
-        rejectAllButton: '.GzLjMd button:nth-child(1)'
-      }
-    }
-  },
-
   elements: {
-    consentModal,
-
```

Comment:
I think we can keep the content in this file, it shows that we can have an element as well as a section with the same name.

Reply:
Sure